From 22644b88821265535b5781d4cd327b11e82c0878 Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Mon, 24 Oct 2022 14:55:35 +0700
Subject: [PATCH] Revert "arm64: dts: renesas: cat875: Add EtherAVB internal
 tx/rx delay"

This reverts commit 7d1a4bf54cb747a1b0a57308bc46a015752a7814.
Phy mode of CAT875 is changed to "rgmii-id" so that no need to set
tx/rx-internal-delay-ps properties anymore.

Signed-off-by: hienhuynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/cat875.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/cat875.dtsi b/arch/arm64/boot/dts/renesas/cat875.dtsi
index e507d651ef0a..8ce15d7a7f3a 100644
--- a/arch/arm64/boot/dts/renesas/cat875.dtsi
+++ b/arch/arm64/boot/dts/renesas/cat875.dtsi
@@ -19,8 +19,6 @@ &avb {
 	renesas,no-ether-link;
 	phy-handle = <&phy0>;
 	phy-mode = "rgmii-id";
-	rx-internal-delay-ps = <1800>;
-	tx-internal-delay-ps = <2000>;
 	status = "okay";
 
 	phy0: ethernet-phy@0 {
-- 
2.38.1

